SECURITY.md

```diff
@@ -13,18 +13,36 @@ Each GeoServer release is supported with bug fixes for a year, with releases mad
 This approach provides ample time for upgrading ensuring you are always working with a supported GeoServer release.
 
 If your organisation is making use of a GeoServer version that is no longer in use by the community all is not lost.
-You can volunteer on the developer list to make additional releases, or engage with one of our [Commercial Support](http://geoserver.org/support/) providers.
+You can volunteer on the developer list to make additional releases, or engage with one of our
+[Commercial Support](http://geoserver.org/support/) providers.
 
 ## Reporting a Vulnerability
 
 If you encounter a security vulnerability in GeoServer please take care to report in a responsible fashion:
 
-* Keep exploit details out of mailing list and issue tracker
-* Send details to geoserver-security@lists.osgeo.org which is monitored by volunteers
-* GitHub [security](https://github.com/geoserver/geoserver/security) page for *Private vulnerability reporting*
-* Be prepared to work with Project Steering Committee (PSC) members on a solution
-* Keep in mind PSC members are volunteers and an extensive fix may require fundraising / resources
+* Keep exploit details out of public mailing lists and issue tracker
+* Send details to volunteers on private geoserver-security@lists.osgeo.org mailing list; or via
+  GitHub [security](https://github.com/geoserver/geoserver/security) page using *Private vulnerability reporting*
+* Be prepared to work with geoserver-security email list volunteers on a solution
+* Keep in mind particpants are volunteering their time, an extensive fix may require fundraising / resources
 
-Please send a mail directly to geoserver-security@lists.osgeo.org (moderated list with no possibility to subscribe, please just send directly to the address, the mail will be evaluated and eventually posted) and provide information about the security issue you might have found there.
+Please send a mail directly to geoserver-security@lists.osgeo.org  and provide information
+about the security vulnerability you might have found there. This is a moderated list:
+send directly to the address; your email will be moderated; and eventually shared with volunteers.
+The volunteer nature of geoserver-security list means that there is no expected response time. 
```

Comment:
But the list above says one can also use the Github path?

Reply:
Yeah I don’t know how to phrase it as an option .. any ideas?

Reply:
It's just that first you present options, but in this list it mandates to start with a mail to the security list instead. The process is then not clear.